setup.py

```diff
@@ -279,17 +279,14 @@ def initialize_options(self):
 
         for root, dirs, files in os.walk('pandas'):
             for f in files:
-                if f in self._clean_exclude:
-                    continue
-
-                # XXX
-                if 'ujson' in f:
```

Comment:
can't remove ujson from this as these are part of the ``src/ujson/lib`` dir.

maybe directly specify ALL of the *.c files by dir and filename (or just dir) for the exclusions